Code/Tools/ProjectManager/Source/FormOptionsWidget.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QWidget>
+#endif
+
+QT_FORWARD_DECLARE_CLASS(QCheckBox)
+QT_FORWARD_DECLARE_CLASS(QFrame)
+
+namespace AzQtComponents
+{
+    class CheckBox;
+}
+
+namespace O3DE::ProjectManager
+{
+    class FormOptionsWidget : public QWidget
+    {
+        Q_OBJECT
+
+    public:
+        FormOptionsWidget(const QString& labelText,
+                          const QStringList& options,
+                          const QString& allOptionsText,
+                          const int optionItemSpacing = 24,
+                          QWidget* parent = nullptr);
+
+        void clear();
+
+        void enable(const QString& option);
+
+        void enable(const QStringList& options);
+
+        void disable(const QString& option);
+
+        void disable(const QStringList& options);
+
+        void enableAll();
+
+        QStringList getOptions() const;
+
+    private:
+        int getCheckedCount() const;
```

Comment:
These member function names do not match our style guide for member function names - they should all start with upper case letters.